### PR TITLE
Fixes #24343 - Update deprecations for 1.20

### PR DESCRIPTION
**Pull request by tbrisker**
Deprecated methods were removed. Two-pane leftovers were cleaned up.
Deprecation messages have been updated to the correct version number.

### CODE REVIEW

**Review by ohadlevy — CHANGES_REQUESTED** [review]
I see alot more code related to two-pane - have a look at https://github.com/theforeman/foreman/search?q=two-pane&unscoped_q=two-pane

**Comment by tbrisker** [comment]
Updated to remove all two-pane leftovers (hopefully)

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ class ApplicationController < ActionController::Base
   before_action :welcome, :only => :index, :unless => :api_request?
   prepend_before_action :allow_webpack, if: -> { Rails.configuration.webpack.dev_server.enabled }
   around_action :set_timezone
-  layout :display_layout?
+  layout "application"
```
Comment (ohadlevy) [inline app/controllers/application_controller.rb]: That's the default, you can omit the layout call.

**Review by ohadlevy — CHANGES_REQUESTED** [review]
tested and looks good, minor nitpick inside, could you also update the commit message? thanks

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (tbrisker) [reply app/controllers/application_controller.rb]: thanks, updated

**Review by ohadlevy — APPROVED** [review]

**Comment by tbrisker** [comment]
Test failure unrelated, care to pull the trigger @ohadlevy ?

**Comment by ohadlevy** [comment]
thanks @tbrisker

**Pull request merged**